nose_parameterized/parameterized.py

```diff
@@ -211,11 +211,53 @@ def expand(cls, input, testcase_func_name=None):
             >>>
             """
 
-        def parameterized_expand_wrapper(f):
+        def doc_expansion(f, args, kwargs, is_method):
+            offset = 1 if is_method else 0
+            vns = inspect.getargspec(f)[0][offset:]
```

Comment:
Okay.

I'm not sure why I added this here - I suspect I was running into an exception locally, although I'm using 2.7 and 3.4 here. I'll have to dig into it and let you know.